collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
@zouyee 
client-go `v4.0.0-beta.0` branch only has [a function](https://github.com/kubernetes/client-go/blob/v4.0.0-beta.0/pkg/api/helper/helpers.go#L549) which use `k8s.io/client-go/pkg/api.PersistentVolumeClaim`. In kube-state-metrics collectors, we all use types under `k8s.io/client-go/pkg/api/v1` package, types are mismatch.